components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -299,6 +407,45 @@ export function CreateWorkspacePage() {
     );
 }
 
+function RememberOptions(params: {
+    disabled?: boolean;
+    checked: boolean;
+    workspaceContext?: WorkspaceContext | null;
+    onChange: (checked: boolean) => void;
+}) {
+    const { disabled, checked, workspaceContext, onChange } = params;
+
+    const label =
+        "Always use these options for " +
+        (CommitContext.is(workspaceContext)
+            ? workspaceContext.repository.owner + "/" + workspaceContext.repository.name
+            : "this repository") +
+        ".";
+
+    return (
+        <>
+            <div className={"w-full flex justify-center mt-3 px-8 mx-2"}>
+                <CheckboxInputField
+                    label={label}
+                    checked={checked}
+                    disabled={disabled}
+                    topMargin={false}
+                    onChange={onChange}
+                />
+            </div>
+            <div className={"w-full flex justify-center px-8 mx-2"}>
+                <p className="text-gray-500 text-sm">
+                    You can reset all remembered options in{" "}
+                    <Link to={settingsPathPreferences} className="gp-link">
+                        preferences
+                    </Link>
+                    .
+                </p>
+            </div>
```

Comment:
**suggestion**: Sounds skippable and easy to guess that clearing the defaults would live under user preferences. Thoughts?

```suggestion
```

Reply:
* I'd like to keep the line saying they can undo this in their preferences. 
* I agree the option looks better under the button.
* Also, removing the repository name will make width consistent and avoid and wrapping/clipping issues. 

<img width="511" alt="image" src="https://user-images.githubusercontent.com/30455/235775720-57549945-5670-4373-bf41-bb34857e2682.png">